pkg/repositories/models/project.go

```diff
@@ -3,7 +3,7 @@ package models
 type Project struct {
 	BaseModel
 	Identifier  string `gorm:"primary_key"`
-	Name        string // Human-readable name, not a unique identifier.
+	Name        string `gorm:"type:varchar(64)"` // Human-readable name, not a unique identifier.
```

Comment:
can we leave the underlying db model as is? i'd like to make validation part of the admin business logic layer so that if we need to change the limits we can easily do that without requiring a db op